lib/config.ml.in

```diff
@@ -23,5 +23,6 @@ let package_version_full = "@PACKAGE_VERSION_FULL@"
 let prefix = "@prefix@"
 let datadir = prefix ^ "/share"
 let host_cpu = "@host_cpu@"
+let nbdkit = if "@NBDKIT@" = "" then None else Some "@NBDKIT@"
```

Comment:
I think this should just be a plain string, `"@NBDKIT@"`.  That makes everything else much less complicated.